### ISSUE TITLE
Question about Muse and ecommerce, marketplace

### ISSUE THREAD

**Issue by ghost** [musetowordpress/muse-to-wordpress#2]
Hi,

First of all, I would like to express my gratitude at what you are doing. Muse to wordpress gives me the best of both worlds.

I was just wondering if it is possible to do a c2c marketplace in my website, designed in muse, exported to wordpress using muse to wordpress. Then using WooCommerce and then WooCommerce Marketplace plugin to create a multi - vendor marketplace like Amazon or etsy

I am a complete newbie and beyond muse, I don't have any experience with web design and don't know coding.

So is it possible to a. host an online e-commerce store with muse and wooCommerce? What pages do I need to create in Muse, where do I upload them to, and what do I do with them on WordPress?

Then, how do I integrate that with another plugin such as WooCommerce Marketplace to create a marketplace? If that particular plugin doesn't work, is there another one that will do the job? I don't mind if its paid. Then, once again, what pages do I need to create and how to set them up?

Sorry if these questions could be very easily answered but I need an answer and I have very little experience. 

If you guys are able to solve this problem, I am one hundred percent becoming a MWuser. I am only 14 and if I am able to create a fully functioning marketplace with no coding, that is truly mind - boggling.

You guys are doing an amazing job, keep it up!!!

P.S. My website is OfficeHome.com.au if it is necessary to check the url.

**Comment by mwuse** [comment]
Yes, using WooCommerce it's possible, but require to understand hierarchy to create the right pages.
Integrate WooCommerce will require a bit coding or you can use our theme and plugin made for MTW: https://musetowordpress.com/product/dr-woo-revolution/
